pallets/ddc-verification/src/lib.rs

```diff
@@ -64,6 +64,9 @@ pub mod pallet {
 	const STORAGE_VERSION: frame_support::traits::StorageVersion =
 		frame_support::traits::StorageVersion::new(0);
 
+	const RESPONSE_CODE: u16 = 200;
```

Comment:
it's http status code, right?

200 stands for success, might be it make sense to include 'success' in the constant name?